requirements.txt

```diff
@@ -5,6 +5,7 @@ gitpython==3.1.27
 isort==5.13.2
 jupyter==1.0.0
 jinja2==3.1.2
+lammps
```

Comment:
Will this install lammps when we set up the project? I think we need lammps in the environment, but it shouldn't be installed by our project setup: it may be available already on the cluster, we wouldn't want to reinstall a conda version ourselves...